### ISSUE TITLE
Members no son clickables

### ISSUE THREAD

**Issue by mdelapenya** [gdgtoledo/web#62]
¿Cómo se ve el perfil de la membresía? Las imágenes/texto no son clickables, y manipulando la URL para llegar a `/members/nombre-miembro` lleva a una página dónde sólo se ve el título y el nombre.

Parece que los metadatos no se cargan en la plantilla HTML/markdown

<img width="1432" alt="Screenshot 2019-09-18 at 16 29 22" src="https://user-images.githubusercontent.com/951580/65157766-8a4c0000-da31-11e9-9430-16c569131642.png">

¿Alguna sugerencia?

**Comment by antoniocrevel** [comment]
Habría que añadir `<a href="{{site.baseurl}}/members/{{ member.url}}">...</a>` a la foto o al nombre (o a ambos) de cada miembro de la página `web/_includes/members.html`, y en el layout `web/_layouts/member.html` habría que incluir el resto de datos que recogemos en el _front matter_ de cada miembro, el cual, si quiere poner su bio en el _content_, también se mostraría.

**Comment by antoniocrevel** [comment]
@javierlopezdeancos , ¿estás haciendo algo de lo que comento arriba en la issue #8 que tienes `In Progress`?

**Comment by javierlopezdeancos** [comment]
EhhHhHHhhHHh

esto no es un bug, el comportamiento es lo esperado en el MVP

Esa feature aún no está.

Se podria definir en una nueva meta issue

- [ ] crear template para miembro.
- [ ] linkar items en lista de miembros con sus perfiles.

**Comment by javierlopezdeancos** [comment]
Lo explica muy bien @antoniocrevel en la issue que se quedó duplicada al abrir @mdelapenya ésta. Gracias @antoniocrevel  por cerrarla y hacer el manejo de esto.

Dicho esto, lo explicaba muy bien @antoniocrevel 

```
¿Creamos entonces la página <gdg-web>/layouts/members.html accesible desde <gdg-web>/members/ para que muestre una lista de los miembros añadidos al site?

Los miembros deberían estar ordenados:

Primero deberían salir los organizadores del GDG (se propone añadir al front matter una propiedad organizer: true)
Después el resto ordenados alfabéticamente
Para cada miembro se mostrará:

Nombre {{page.name}} {{page.last_name}}
Foto {{page.image}}
Cuenta de Twitter y GitHub {{page.twitter}} {{page.github}}
```


Asi que, que os parece si organizamos la información y creamos la meta con las issues correspondientes, las añadimos al proyecto y las metemos en un milestone?

hace falta una videollamada o lo podemos hacer async?

**Comment by javierlopezdeancos** [comment]
Otra cosa, voy a poner `medium` la prioridad de las cosas prioritarias que no sean bugs.

**Comment by javierlopezdeancos** [comment]
> @javierlopezdeancos , ¿estás haciendo algo de lo que comento arriba en la issue #8 que tienes `In Progress`?

no, la tengo asignada pero no está in progress ya que no está en ningún `project` ni `milestone`

**Comment by mdelapenya** [comment]
Agree a todo :)

Sí que es cierto que, como developer ocasional del proyecto, meter datos en el frontmatter del member y luego no verlos es raro. Por eso marqué como bug, pero me parece bien moverlo a question y para otra story 👍
